docs/source/ext/github_tools.py

```diff
@@ -42,6 +42,10 @@
 
 
 def fetch_url(url):
+    '''This was pointed out as a Security issue in bandit.
+    please look at issue #355,
+    we fixed it, but the bandit warning might remain,
+    need to suppress it manually (just ignore it)'''
```

Comment:
The docstring is incomplete. Please, use double-quotes and follow the standard of our docstring. As an example, you can look [here](https://github.com/fury-gl/fury/blob/master/fury/colormap.py#L302-L324)

Reply:
You did not address my comment above. Please, look at [the example](https://github.com/fury-gl/fury/blob/master/fury/colormap.py#L302-L324) cited above and complete the docstring. 
As suggested, the text should be in a `NOTE`